### PR TITLE
the chaplain can now choose a weakened Eutactic energy sword as a holy weapon

### PR DESCRIPTION
**Pull request by silicons**
THIS IS A BAD IDEA @deathride58

### CODE REVIEW

**Comment by deathride58** [comment]
this is a good idea

File: modular_citadel/code/game/objects/items/melee/eutactic_blades.dm [hunk 1]
```diff
@@ -117,6 +117,15 @@
 	desc = "The Non-Eutactic Blade utilizes a hardlight blade that is dynamically 'forged' on demand to create a deadly sharp edge that is unbreakable. This one seems to have a damaged handle and misaligned components, causing the blade to be unstable at best"
 	force_on = 15 //As strong a survival knife/bone dagger
 
+/obj/item/melee/transforming/energy/sword/cx/chaplain
+	name = "divine lightblade"
+	force_on = 20		//haha i'll regret this
+	block_chance = 50
```
Comment (Trilbyspaceclone) [inline modular_citadel/code/game/objects/items/melee/eutactic_blades.dm:136]: This is flat out better then claymore and other weapons

Comment (Trilbyspaceclone) [reply modular_citadel/code/game/objects/items/melee/eutactic_blades.dm:136]: Normal null rod is 18
And the highest block you get is 30 with the claymore

**Review by deathride58 — DISMISSED** [review]
deus vult yolo swag

File: modular_citadel/code/game/objects/items/melee/eutactic_blades.dm (hunk 1 above)
Comment (silicons) [reply modular_citadel/code/game/objects/items/melee/eutactic_blades.dm:136]: think this'll be fine.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**